getSelects.php

```diff
@@ -0,0 +1,54 @@
+<?php
+    //This file is for getting user information for switching session event data such as location, time and day. 
+    //pdo.php contains the pdo variable.
+    include './pdo.php';
+    //Gets possible day selections
+    echo "<h2>Change Session:</h2><br>";
+    echo "<div id='dayselectdiv'>";
+        echo "<h3>Select Day:</h3>";
+        //Gets the days to select from. The form attribute allows for drop down menu information to be passed when user submits data.
+        echo "<select id='sessionselect' name='sessionday' form='setEventForm'>";
+            echo "<option value='saturday'>Saturday</option>";
+            echo "<option value='sunday'>Sunday</option>";
+        echo "</select><br>";
+    echo "</div>";
+
+    //Gets the possible rooms to select from";
+    echo "<div id='roomselectdiv'>";
+        echo "<h3>Rooms:</h3>";
```

Comment:
IMO this should just be whatever user decides the room number should be, but i'm not really opposed to this method as well considering how hacky we're allowed to be with this project.

Reply:
Yeah I agree, this was pretty hacky. Wanted to restrict input from the user for security reasons? But I guess that doesn't make sense since they could just query the end point.

Reply:
No biggies, i've warmed up to the idea. We'll just state this in our assumptions